test/scale/scale_tester.go

```diff
@@ -235,7 +230,7 @@ func (t *PodSchedulingOnFailoverScaleTest) Run(client clientset.Interface, names
 			_, err := client.AppsV1().StatefulSets(ss.Namespace.Name).UpdateScale(context.TODO(), ss.Statefulset.Name, newScale, metav1.UpdateOptions{})
 			framework.ExpectNoError(err)
 			time.Sleep(30 * time.Second)
-			ss.WaitForPodReady()
+			_ = ss.WaitForPodReady()
```

Comment:
Should we check and fail on error here?

Reply:
These tests are currently not running, and Akash is working on updating these as a part of #350. I didn't want to call `framework.ExpectNoError(err)` in a goroutine here but it seems like all the goroutines in this test do that, so I'll add it here as well.